test/system/700-play.bats

```diff
@@ -82,6 +82,8 @@ RELABEL="system_u:object_r:container_file_t:s0"
     run_podman version --format "{{.Server.Version}}-{{.Server.Built}}"
     run_podman image exists localhost/podman-pause:$output
 
+    run_podman ps
```

Comment:
Is this being used in this test.

Reply:
That's a (harmless) debugging artifact.  I can remove it in a follow-up PR or in this one in case I need to repush.

Thank you for catching!